src/extensions/components/ExtenionItem/components/InstalledBadge.tsx

```diff
@@ -0,0 +1,19 @@
+import { SUCCESS_ICON_COLOR } from "@dashboard/colors";
+import { Text } from "@saleor/macaw-ui-next";
+import React from "react";
+import SVG from "react-inlinesvg";
+import { useIntl } from "react-intl";
+
+import CheckIcon from "../../../../../assets/images/check.svg";
+import { messages } from "../../../messages";
```

Comment:
suggestion: these imports are super long, do we have maybe an alias for that?

Reply:
I prefer to keep all import within module as relative